common/src/main/java/whocraft/tardis_refined/common/tardis/manager/AestheticHandler.java

```diff
@@ -32,6 +32,7 @@ public class AestheticHandler extends BaseHandler {
     private ResourceLocation shellTheme = ShellTheme.FACTORY.getId();
     private ShellPattern shellPattern = ShellPatterns.DEFAULT;
     private HumEntry currentHum = TardisHums.getDefaultHum();
+    public static final double REQUIRED_SHELL_FUEL = 15; // Amount of fuel required to change the shell todo - make non static
```

Comment:
Mixed feelings about this being here

Reply:
unsure where it should go but for now ill move it to `TardisExteriorManager` makes a smidge more sense i guess